1-Introduction/3-fairness/translations/README.es.md

```diff
@@ -19,61 +19,61 @@ En esta lección, será capaz de:
 
 ## Prerrequisitos
 
-Como un prerrequisito, por favor toma el Path de aprendizaje "Responsible AI Principles" y mira el video debajo en el tema:
+Como un prerrequisito, por favor toma el curso "Responsible AI Principles" y mira el vídeo debajo sobre el tema:
 
-Aprende más acerca de la AI responsable siguiendo este [Path de aprendizaje](https://docs.microsoft.com/learn/modules/responsible-ai-principles/?WT.mc_id=academic-15963-cxa)
+Aprende más acerca de la AI responsable siguiendo este [curso](https://docs.microsoft.com/learn/modules/responsible-ai-principles/?WT.mc_id=academic-15963-cxa)
```

Comment:
Since this is Spanish, you could update the link to force Spanish and update the title to the existing translation:
```suggestion
Aprende más acerca de la AI responsable siguiendo este [curso](https://docs.microsoft.com/es-es/learn/modules/responsible-ai-principles/?WT.mc_id=academic-15963-cxa)
```

Reply:
Ok, I will change what you mentioned and make a PR as soon as I can.

Thank you for the feedback!